model_compression_toolkit/core/common/graph/base_graph.py

```diff
@@ -907,11 +908,23 @@ def disable_fused_nodes_activation_quantization(self):
         """
```

Comment:
please change this function name to "override_fused_node_activation_quantization_candidates"